vendor/github.com/zmap/zlint/util/gtld_map.go

```diff
@@ -0,0 +1,7830 @@
+// Code generated by go generate; DO NOT EDIT.
+// This file was generated by zlint-gtld-update.
+
+/*
+ * ZLint Copyright 2018 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package util
+
+var tldMap = map[string]GTLDPeriod{
+	"aaa": {
```

Comment:
same questions about allocation on init of all binaries, frequency of updates, and implications of being outdated